tests/test_pytest_plugin.py

```diff
@@ -116,29 +116,29 @@ def test_noncoro():
     assert True
 
 
-async def test_client_failed_to_create(test_client):
+async def test_failed_to_create_client(aiohttp_client):
 
     def make_app(loop):
         raise RuntimeError()
 
     with pytest.raises(RuntimeError):
-        await test_client(make_app)
+        await aiohttp_client(make_app)
 
 
-async def test_custom_port_test_client(test_client, unused_port):
-    port = unused_port()
-    client = await test_client(create_app, server_kwargs={'port': port})
+async def test_custom_port_aiohttp_client(aiohttp_client, aiohttp_unused_port):
+    port = aiohttp_unused_port()
```

Comment:
@asvetlov 
Why use it at all if it's possible to instruct app to bind 0 and read the allocated port when needed? That fixture unbinds from port, meaning it might be intercepted by smth else as a result of race condition.
I'd go for `SO_REUSEPORT` or socket activation in order to make it more reliable.

Reply:
The approach is widely used by asyncio tests, I just borrowed the idea.

port 0 may work but the change is definitely out of the PR scope.
Feel free to create a new PR if needed.

Reply:
Yea, I understand. It's just that I've been concerned about it during some time now. And decided to shout out about possible flaky behavior in test flow. I would at least not expose this as a public API to end-users, encouraging them to use port=0

Reply:
As first step we should get rid of the function in aiohttp test suite itself.
Maybe in next releases?